docs/static/breaking-changes-90.asciidoc

```diff
@@ -13,3 +13,26 @@ Include:
 * user impact/decription/value prop
 * link to relevant docs for more information
 
+[discrete]
+[[Removal-of-Deprecated-SSL-Settings]]
+===== Removal of Deprecated SSL Settings
+
+The following plugins have removed previously deprecated SSL settings:
+
+[discrete]
+[[logstash-output-http]]
+====== logstash-output-http
+
+[cols="<,<",options="header",]
+|=======================================================================
+|Setting|Replaced by
+| cacert |<<plugins-outputs-http-ssl_certificate_authorities>>
+| client_cert |<<plugins-outputs-http-ssl_certificate>>
+| client_key |<<plugins-outputs-http-ssl_key>>
+| keystore |<<plugins-outputs-http-ssl_keystore_path>>
+| keystore_password |<<plugins-outputs-http-ssl_keystore_password>>
+| keystore_type |<<plugins-outputs-http-ssl_keystore_password>>
+| truststore |<<plugins-outputs-http-ssl_truststore_path>>
+| truststore_password |<<plugins-outputs-http-ssl_truststore_password>>
+| truststore_type |<<plugins-outputs-http-ssl_truststore_type>>
+|=======================================================================
```

Comment:
```suggestion
[discrete]
[[ssl-settings-9.0]]
===== Changes to SSL settings in {ls} plugins

We've removed deprecated SSL settings in some {ls} plugins, and have replaced them with updated settings.
If your plugin configuration contains any of these obsolete options, the plugin may fail to start.

**Plugins with changes to SSL settings** 

[discrete]
[[output-http-ssl-9.0]]
.<<plugins-outputs-http,logstash-output-http>>

[%collapsible]
====

[cols="<,<",options="header",]
|=======================================================================
|Setting|Replaced by
| cacert |<<plugins-outputs-http-ssl_certificate_authorities>>
| client_cert |<<plugins-outputs-http-ssl_certificate>>
| client_key |<<plugins-outputs-http-ssl_key>>
| keystore |<<plugins-outputs-http-ssl_keystore_path>>
| keystore_password |<<plugins-outputs-http-ssl_keystore_password>>
| keystore_type |<<plugins-outputs-http-ssl_keystore_password>>
| truststore |<<plugins-outputs-http-ssl_truststore_path>>
| truststore_password |<<plugins-outputs-http-ssl_truststore_password>>
| truststore_type |<<plugins-outputs-http-ssl_truststore_type>>
|=======================================================================

====
```

Reply:
Let's consider setting this up as expandable/collapsible sections to help users focus on the plugins they're most interested in.

<img width="981" alt="Screenshot 2024-11-21 at 12 02 53 PM" src="https://github.com/user-attachments/assets/725efe86-b5d5-4dd7-a65c-e693bded13dc">

Reply:
Other general guidelines:
* Make heading anchors more specific (examples in lines 17 and 26).
* Be sure to describe user impact (line 21). 
* Include link to related docs if appropriate (line 27)
* Follow Elastic docs standard for sentence style headings (line 18).

Reply:
Big fan of the collapsible sections - there's going to be a lot of similar content here, so anything that conveys everything we need without dominating the whole breaking changes docs works for me.

Other suggestions all make sense to me too